docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml

```diff
@@ -281,35 +281,16 @@ myelk-kibana LoadBalancer 10.110.129.199 localhost 5601:31465/TCP 68m
         values file and we provide an example command line that inserts that
         values file into your deployment.</para>
 
-        <para>These are the Elastic4HPCCLogs values:</para>
-
-        <programlisting># Configures HPCC logAccess to target elastic4hpcclogs deployment
-global:
-  logAccess:
-    name: "Elastic4HPCCLogs"
-    type: "elasticstack"
-    connection:
-        protocol: "http"
-        host: "elasticsearch-master.default.svc.cluster.local"
-        port: 9200
-    logMaps:
-      - type: "global"                             #These settings apply to all log mappings
-        storeName: "hpcc-logs*"                    #Logs are expected to be housed in ES inde
-        searchColumn: "message"                    #The 'message' field is to be targeted for
-        timeStampColumn: "@timestamp"              #The '@timestamp' field contains time log 
-      - type: "workunits"                          #Search by workunits specific log mapping
-        storeName: "hpcc-logs*"                    # Only needed if differs from global.store
-        searchColumn: "hpcc.log.jobid"             # Field containing WU information
-      - type: "components"                         #Search by components specific log mapping
-        searchColumn: "kubernetes.container.name"  # Field containing container information
-      - type: "audience"                           #Search by audience specific log mapping
-        searchColumn: "hpcc.log.audience"          # Field containing audience information
-      - type: "class"                              #Search by log class specific log mapping
-        searchColumn: "hpcc.log.class"             # Field containing log class information
-      - type: "instance"                           #Search by log source instance specific ma
-        searchColumn: "kubernetes.pod.name"        # Field containing source instance informa
-      - type: "host"                               #Search by log source host specific mappin
-        searchColumn: "kubernetes.node.hostname"   # Field containing source host information</programlisting>
+        <para>These are the Elastic4HPCCLogs values can be found on the
```

Comment:
wording should change here since you're no longer providing the values. You should introduce the linked file as suggested configuration values for enabling logaccess against a managed instance of elastic4hpcclogs.